psi4/driver/procrouting/proc.py

```diff
@@ -1299,6 +1300,16 @@ def scf_helper(name, post_scf=True, **kwargs):
     if ref_wfn is not None:
         raise ValidationError("Cannot seed an SCF calculation with a reference wavefunction ('ref_wfn' kwarg).")
 
+    # decide if we keep the checkpoint file
+    _chkfile = kwargs.get('write_orbitals', True)
+    write_checkpoint_file = False
+    if isinstance(_chkfile, str):
+        write_checkpoint_file = True
+        filename = kwargs.get('write_orbitals')
+        core.set_local_option("SCF", "ORBITALS_WRITE", filename)
+    elif isinstance(_chkfile, bool) and _chkfile:
```

Comment:
Isn't this just checking `_chkfile == True`?

Reply:
Nearly. It can be replaced by `elif _chkfile is True:`, which is only true if `_chkfile` is `True` (as opposed a comparison using `==` which would evaluate `_chkfile`).

Reply:
`elif _chkfile is True` is neat.